modules/azure/postgresql.go

```diff
@@ -0,0 +1,135 @@
+package azure
+
+import (
+	"context"
+
+	"github.com/Azure/azure-sdk-for-go/services/postgresql/mgmt/2017-12-01/postgresql"
+	"github.com/gruntwork-io/terratest/modules/testing"
+	"github.com/stretchr/testify/require"
+)
+
+// GetPostgreSQLServerClientE is a helper function that will setup a postgresql server client.
+func GetPostgreSQLServerClientE(subscriptionID string) (*postgresql.ServersClient, error) {
+	// Validate Azure subscription ID
+	subscriptionID, err := getTargetAzureSubscription(subscriptionID)
+	if err != nil {
+		return nil, err
+	}
+
+	// Create a postgresql server client
+	postgresqlClient := postgresql.NewServersClient(subscriptionID)
+
+	// Create an authorizer
+	authorizer, err := NewAuthorizer()
+	if err != nil {
+		return nil, err
+	}
+
+	// Attach authorizer to the client
+	postgresqlClient.Authorizer = *authorizer
+
+	return &postgresqlClient, nil
+}
+
+// GetPostgresqlServer is a helper function that gets the server.
+// This function would fail the test if there is an error.
+func GetPostgresqlServer(t testing.TestingT, resGroupName string, serverName string, subscriptionID string) *postgresql.Server {
```

Comment:
Can you make all the functions consistent and use `PostgreSQL` instead of `Postgresql`?

Reply:
@omichels could you please update the PR to resolve the last comment, or give me access to your repo and I would do it.

Reply:
"all functions consistent using PostgreSQL" 
commit: 00727ae